Zend/zend_vm_def.h

```diff
@@ -8883,6 +8883,11 @@ ZEND_VM_COLD_CONST_HANDLER(190, ZEND_COUNT, CONST|TMPVAR|CV, UNUSED)
 
 	SAVE_OPLINE();
 	op1 = GET_OP1_ZVAL_PTR_UNDEF(BP_VAR_R);
+
+	if ((OP1_TYPE & (IS_VAR|IS_CV)) != 0 && Z_TYPE_P(op1) == IS_REFERENCE) {
```

Comment:
This check should stay where it was before (on the slow path).

Reply:
Oh that's why it was there, will revert this bit then